static/src/components/task_browse/assign_worker.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="form-horizontal">
+    <div class="form-group">
+      <div class="col-sm-12">
+        <GigSpinner v-if="waiting" />
+        <div>
+          <label class="typo__label">Assign users</label>
+          <multiselect
+            v-model="addUserValues"
+            placeholder="Search"
+            label="fullname"
+            track-by="email"
+            :options="allUserList"
+            :multiple="true"
+          />
+        </div>
+        <br>
+        <div>
+          <label class="typo__label">Remove users</label>
+          <multiselect
+            v-model="removeUserValues"
+            placeholder="Search"
+            label="fullname"
+            track-by="email"
+            :options="assignedUserList"
+            :multiple="true"
+          />
+        </div>
+      </div>
+    </div>
+    <p>
+      Note:
+      Type in users to assign them to the selected tasks
+    </p>
+  </div>
+</template>
+
+<style src="vue-multiselect/dist/vue-multiselect.min.css"></style>
+<script>
+import { mapGetters } from 'vuex';
+import Multiselect from 'vue-multiselect';
+import GigSpinner from '../common/gig_spinner.vue';
+
+export default {
+  components: {
+    Multiselect,
+    GigSpinner
+  },
+
+  data () {
+    return {
+      waiting: false,
+      allUserList: [],
+      assignedUserList: [],
+      addUserValues: [],
+      removeUserValues: []
+    };
+  },
+
+  methods: {
+    ...mapGetters([
+      'getSelectedTask',
+      'getFilters',
+      'getCsrfToken'
+    ]),
+
+    initialize (data) {
+      console.log('initialization');
+      console.log(data);
+      this.allUserList = data.all_users || [];
+      this.assignedUserList = data.assign_users || [];
+    },
+
+    getURL () {
+      let path = window.location.pathname;
+      console.log(path);
+      let res = path.split('/');
+      if (res[res.length - 1] !== 'browse') {
+        res = res.splice(0, res.length - 2);
+      }
+      res[res.length - 1] = 'assign-workersupdate';
+      console.log(res.join('/'));
+      return res.join('/');
+    },
+
+    async getData () {
+      let requestData = {
+        taskId: this.getSelectedTask(),
+        filters: JSON.stringify(this.getFilters())
+      };
+      console.log(this.getCsrfToken());
+      try {
+        this.waiting = true;
+        const res = await fetch(this.getURL(), {
+          method: 'POST',
```

Comment:
I think get data expected to make `GET` request and save would make `POST/PUT` request. Any reason why getData not making `GET` request instead of `POST`?

Reply:
I have to use a POST request in order to include body: JSON.stringify(requestData) into the call